DSCResources/MSFT_xNetworkTeam/MSFT_xNetworkTeam.psm1

```diff
@@ -0,0 +1,230 @@
+﻿if (Test-Path "${PSScriptRoot}\${PSUICulture}")
+{
+    Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xNetworkTeam.psd1 -BaseDirectory "${PSScriptRoot}\${PSUICulture}"
+}
```

Comment:
See [notes](https://technet.microsoft.com/en-us/library/hh849919.aspx)  Why don't we just let import-localizeddata do the search?